app/modules/transactions.js

```diff
@@ -139,16 +141,23 @@ export const sendTransaction = ({
       )
     }
 
+    const config = {
+      net,
+      tokensBalanceMap,
+      address: fromAddress,
+      publicKey,
+      privateKey: new wallet.Account(wif).privateKey,
+      signingFunction: isHardwareSend ? signingFunction : null,
+      fees
+    }
+
+    if (url) {
```

Comment:
I would double check neon-js, but I think if url is undefined it won't use it, so I don't see the purpose of this check to add the url, you can just add it to the config above: eg. config = { net, ..., fees, url }